kong/router/atc.lua

```diff
@@ -2,6 +2,7 @@ local _M = {}
 local _MT = { __index = _M, }
 
 
+local isempty= require("table.isempty")
```

Comment:
Could we move this line below `local tb_new` and rename it to `tb_isempty`?

Reply:
I have tested this on my side, it will break the old test cases: https://github.com/Kong/kong/pull/13024#discussion_r1599677152

Reply:
should use `table.nkeys` as `tb_nkeys`.